stable/pgadmin/templates/deployment.yaml

```diff
@@ -56,7 +56,11 @@ spec:
         - name: pgadmin-data
         {{- if .Values.persistence.enabled }}
           persistentVolumeClaim:
+          {{- if .Values.persistence.existingClaim }}
+            claimName: {{ .Values.persistence.existingClaim |quote }}
```

Comment:
Wouldn't this look a bit better?:

claimName: {{ if .Values.persistentVolume.existingClaim }}{{ .Values.persistentVolume.existingClaim }}{{- else }}{{ $fullName }}{{- end }}

Reply:
Would also need a persistentVolume.existingClaim within the README.md then "Persistent Volume existing claim name".

Reply:
Works for me.